src/main/kotlin/Stores/ChannelSubscriptionStore.kt

```diff
@@ -0,0 +1,34 @@
+package Stores
+
+import models.ChannelSubscription
+import io.ktor.http.Parameters
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.insert
+import org.jetbrains.exposed.sql.select
+import org.joda.time.DateTime
+import java.util.*
+
+object ChannelSubscriptionStore {
+    suspend fun createChannelSubscription(params: Parameters): String? {
+        val uuid = UUID.randomUUID().toString()
+
+        DatabaseFactory.dbQuery {
+            ChannelSubscription.insert {
+                it[userId] = params["userUuid"]!!
```

Comment:
all id's right now are uuid's to my knowledge. Let's hold off on specifying "Uuid" and just have "xId"

Reply:
thoughts?

Reply:
sure can switch to userId,ChannelId?  Not sure what is better:o